### PR TITLE
Gir tutorial publish crate

### PR DESCRIPTION
**Pull request by grelltrier**
Added a little hint at the end to publish the crate to crates.io

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Your first should only be part of the other PR. ;)

**Comment by grelltrier** [comment]
How do I remove it?

**Comment by GuillaumeGomez** [comment]
```
git rebase -i HEAD~2
# you are now in a text editor
# remove the line of the commit you want to remove then save and exit
git push -f origin HEAD:gir-tutorial-publish-crate
```

File: docs-src/tutorial/gir_tutorial.md [hunk 1]
```diff
@@ -550,6 +552,7 @@ In case a function is badly generated, you don't **always** need to reimplement
 ## Words of the end
 
 That's it, with this we should be able to generate any GNOME crate you want. Happy coding!
+You can easily publish your generated bindings and the wrapper to crates.io to allow others to use it. It is best to publish both but in separate crates so people can chose the one they want. Publishing crates is surprisingly easy. If this is your first time, you can follow the guide from the [Rust book](https://doc.rust-lang.org/cargo/reference/publishing.html). Crates.io needs additional information in your Cargo.toml. Gir will not override them when you re-generate bindings. Easy right?
```
Comment (sdroege) [inline docs-src/tutorial/gir_tutorial.md:555]: They should be two separate crates (-sys and non-sys) also for technical reasons as only one crate can link to a specific C library in the whole dependency tree.

**Comment by grelltrier** [comment]
I added similar information to the draft pull request to rewrite the gir tutorial so this can be closed

**Pull request closed without merging**